modules/org/infinispan/runtime/added/bin/init_fips_keystore.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# ===================================================================================
+# Init script which sets up certificates in NSSDB for FIPS.
+# ===================================================================================
+
+NSSDB=/etc/pki/nssdb
+KEYSTORE_SECRET=""
+
+ARGS=()
+
+while [ $# -gt 0 ]; do
+  case $1 in
+    -d|--database)
+      NSSDB="$2"
+      shift 2
+      ;;
+    -p|--password)
+      KEYSTORE_SECRET="$2"
+      shift 2
+      ;;
+    -*)
+      echo "Unknown option $1"
+      exit 1
+      ;;
+    *)
+      ARGS+=("$1")
+      shift
+      ;;
+  esac
+done
+
+set -- "${ARGS[@]}"
+
+
+if [ "$#" -eq 0 ]; then
+  echo "Usage: $0 [-d nssdb] path [password]"
+  exit 1
+fi
+
+KEYSTORE_PATH=$1
+
+if [ ! -d "$NSSDB" ]; then
+  echo "Directory $NSSDB does not exist"
+  exit 1
+fi
+
+if [ ! -e "$NSSDB/pkcs11.txt" ]; then
+  echo "Directory $NSSDB does not appear to be a NSS database"
+  exit 1
+fi
+
+CERTIFICATES=$(ls -1 "$KEYSTORE_PATH"/*.crt 2>/dev/null | wc -l)
+if [ "$CERTIFICATES" != 0 ]
+then
+  for CRT in $KEYSTORE_PATH/*.crt; do
+    NAME=$(basename "$CRT" .crt)
+    echo "Converting $NAME.crt/$NAME.key to $NAME.p12"
+    openssl pkcs12 -export -out "$KEYSTORE_PATH/$NAME.p12" -inkey "$KEYSTORE_PATH/$NAME.key" -in "$CRT" -name "$NAME" -password "pass:$KEYSTORE_SECRET"
+  done
+else
+  if [ "$#" -ne 2 ]; then
```

Comment:
As we already have the `-p` option, I think it would be simpler if we just ensure that `KEYSTORE_SECRET` is not empty here